cmd/perunnodecli/contact.go

```diff
@@ -29,21 +29,21 @@ var (
 	contactCmdUsage = "Usage: contact [sub-command]"
 	contactCmd      = &ishell.Cmd{
 		Name: "contact",
-		Help: "Use this command to add/get contacts." + contactCmdUsage,
+		Help: "Use this command to add/get idProvider." + contactCmdUsage,
 		Func: contactFn,
 	}
 
 	contactAddCmdUsage = "Usage: contact add [peer alias] [off-chain address] [comm address] [comm type]"
 	contactAddCmd      = &ishell.Cmd{
 		Name: "add",
-		Help: "Add a peer to contacts." + contactAddCmdUsage,
+		Help: "Add a peer to idProvider." + contactAddCmdUsage,
 		Func: contactAddFn,
 	}
 
 	contactGetCmdUsage = "Usage: contact get [peer alias]"
 	contactGetCmd      = &ishell.Cmd{
 		Name: "get",
-		Help: "Get peer info from contacts." + contactGetCmdUsage,
+		Help: "Get peer info from idProvider." + contactGetCmdUsage,
```

Comment:
[Kept](https://github.com/hyperledger-labs/perun-node/pull/135/commits/0a6e8feef151b888b0131cbb4f0311a8cfa0abdc) peer ID instead of contacts and changed other variable and names according to the context.

Reply:
[Kept](https://github.com/hyperledger-labs/perun-node/pull/135/commits/d7c5500a7b2664abeae8f449eddbbd7d163a6b49) peer ID and changed variables and names according to the context.